docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+  kuzzle:
+    image: kuzzleio/plugin-dev:2
+    command: /run-dev.sh
+    volumes:
+      - './docker/run-dev.sh:/run-dev.sh'
+      - './docker/install-plugins.sh:/install-plugins.sh'
+      - './docker/pm2.json:/config/pm2.json'
+      - '.:/var/app/plugins/enabled/kuzzle-plugin-graphql'
+    cap_add:
+      - SYS_PTRACE
+    ulimits:
+      nofile: 65536
+    sysctls:
+      - net.core.somaxconn=8192
+    depends_on:
+      - redis
+      - elasticsearch
+    ports:
+      - '9229:9229'
+      - '7512:7512'
+    environment:
+      - kuzzle_services__storageEngine__client__node=http://elasticsearch:9200
+      - kuzzle_services__internalCache__node__host=redis
+      - kuzzle_services__memoryStorage__node__host=redis
+      - NODE_ENV=development
+      - DEBUG=kuzzle:plugins
+
+  redis:
+    image: redis:5
+
+  elasticsearch:
+    image: kuzzleio/elasticsearch:7.3.0
```

The `7` tag is always pointing of our latest release of the `kuzzleio/elasticsearch` image (which is `7.4.0` at the time of this writing)
```suggestion
    image: kuzzleio/elasticsearch:7
```